content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx

```diff
@@ -7,11 +7,12 @@ metaDescription: Prisma Client features that are currently in preview.
 
 The following Preview feature flags are available for Prisma Client:
 
-* [`orderByRelations`](../prisma-client/filtering-and-sorting#sort-by-relation-aggregate-preview)
-* [`selectRelationCount`](../prisma-client/aggregation-grouping-summarizing#count-relations)
-* [`orderByAggregateGroup`](../prisma-client/aggregation-grouping-summarizing#order-by-aggregate-group-preview)
-* `nApi` (N-API support)
-* [`microsoftSqlServer`](sql-server)
+* [`orderByRelations`](../prisma-client/filtering-and-sorting#sort-by-relation-aggregate-value-preview) [Submit feedback](https://github.com/prisma/prisma/issues/5438)
+* [`selectRelationCount`](../prisma-client/aggregation-grouping-summarizing#count-relations) [Submit feedback](https://github.com/prisma/prisma/issues/6312)
+* [`orderByAggregateGroup`](../prisma-client/aggregation-grouping-summarizing#order-by-aggregate-group-preview) [Submit feedback][https://github.com/prisma/prisma/issues/6545]
+* `nApi` (N-API support) [Submit feedback](https://github.com/prisma/prisma/issues/6301)
```

Comment:
Are we changing this to Node API? Instead of n-api, as per @janpio talk on monday

Reply:
Would be good, maybe in a separate PR?

Reply:
Already open at https://github.com/prisma/docs/pull/1834